M2/cmake/FindFFI.cmake

```diff
@@ -1,81 +0,0 @@
-# Attempts to discover ffi library with a linkable ffi_call function.
```

Comment:
I'd rather you didn't entirely delete this file, in case it may be useful in the future.

Reply:
How about putting the hash of the commit, which removed it, next to the commented out `find_package(FFI...` ?

Reply:
I'd be surprised if it ever would need resurrection. I can imagine `libffi` getting a `cmake` interface,
then it could be used instead of `pkg_` interface.

Reply:
If your solution doesn't work on some machine, I want to be able to simply uncomment the previous lines and comment out the pkgconfig line without needing to bring back a file with git.

Reply:
OK, done